.github/workflows/tests.yml

```diff
@@ -35,30 +35,37 @@ jobs:
           filters: .github/filters.yaml
 
   lint:
-    name: 'lint (node: ${{ matrix.node }})'
-    needs: [changes]
+    name: 'lint (node: 18)'
```

Comment:
I think this is fine, lint should (in theory) be independent of node version.

However, can we make it 20 while we're at it just so we're on the current version?

Reply:
@innerdvations I would like to but lint (node: 18) is required to run soo then the required to run list needs to change and I am not sure if I can do that.

Reply:
Do you mean getting it past the current CI? If that's the case I can make sure we update on our end to let it through.

Reply:
Then I will also remove the unit backend test for 16.